### ISSUE TITLE
találatokból szűrő

### ISSUE THREAD

**Issue by pisipite** [Code-for-Hungary/agrartamfront#86]
az lehetséges, hogy a találatok egyes részei linkként szűrőként működjenek? pl. a _támogatott entitás_ vagy a _cégcsoport_ kattintásra kiadja az adott entitáshoz/cégcsoporthoz tartozó találatokat.

**Comment by law909** [comment]
`kattintásra kiadja` helyett kiválasztja a szűrőben, aztán majd a user megnyomja a keresést.

**Comment by law909** [comment]
Az eredménytáblában link legyen a cégcsoport/entitás? neve, amivel a szűrőben kiválasztom a cégcsoportot/entitást.

**Comment by pisipite** [comment]
igen, az alábbiak közül ha valamelyikre kattint a felhasználó, akkor azt válassza ki a fenti szűrők közül (adja hozzá az aktuális szűréshez, tehát szűkítse a keresést).

- Év
- Település neve
- Cégcsoporthoz tartozó nyertes
- Cégcsoport
- Alap
- Forrás
- Jogcím
